src/parse.h

```diff
@@ -295,6 +295,9 @@ struct net_definition {
     NetplanNetDefinition* vlan_link;
     gboolean has_vlans;
 
+    /* carrier */
+    gboolean configure_without_carrier;
```

Comment:
This needs to move to the very end of the `net_definition` struct, to avoid breaking the ABI/memory layout on upgrades.